asistente_ladm_col/gui/controlled_measurement_dialog.py

```diff
@@ -114,31 +129,151 @@ def accept_dialog(self):
+            else:
+                features, not_features = self.time_filter(input_features, input_features.getFeatures("\"{}\"={}".format("belongs_to_group", i)), idx, time_tolerance)
+                features = [f for f in features]
+                not_features = [f for f in not_features]
+                fg = len(features)
+                if fg > 1:
+                    time_layer.startEditing()
+                    time_layer.dataProvider().addFeatures(features)
+                    [time_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("trusty"), "True") for feat in
+                     time_layer.getFeatures()]
+                    time2_layer.startEditing()
+                    time2_layer.dataProvider().addFeatures(not_features)
+                    [time2_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("trusty"), "False") for feat in
+                     time2_layer.getFeatures()]
+                    #[time_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("belongs_to_group"), fg) for
+                    # feat in time_layer.getFeatures()]
+                    time_layer.updateFields()
+                else:
+                    time2_layer.startEditing()
+                    time2_layer.dataProvider().addFeatures(features)
+                    [time2_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("trusty"), "False") for feat in
+                     time2_layer.getFeatures()]
+                    [time2_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("belongs_to_group"), None) for
+                     feat in time2_layer.getFeatures()]
+                    time2_layer.dataProvider().addFeatures(not_features)
+                    [time2_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("trusty"), "False") for feat in
+                     time2_layer.getFeatures()]
+                    time2_layer.updateFields()
+                [time2_layer.changeAttributeValue(feat.id(), feat.fields().indexOf("belongs_to_group"), None) for feat in
+                 time2_layer.getFeatures()]
+                time2_layer.updateFields()
+                time_layer.commitChanges()
+                time2_layer.commitChanges()
+
+        new_layer.startEditing()
+        new_layer.dataProvider().addFeatures(time_layer.getFeatures())
+        new_layer.dataProvider().addFeatures(time2_layer.getFeatures())
+        new_layer.commitChanges()
+        QgsProject.instance().addMapLayer(new_layer)
+        return new_layer
+
+    def copy_attribs(self, sourceLYR, name="Previous Average Points"):
+        destLYR = QgsVectorLayer("Point?crs=EPSG:3116", name, "memory")
+        destLYR.startEditing()
+        destLYR.dataProvider().addAttributes(sourceLYR.fields())
+        destLYR.addAttribute(QgsField("trusty", QVariant.String))
+        destLYR.updateFields()
+        return destLYR
+
+    def time_filter(self, layer, features, idx, time_tolerance):
```

Comment:
Again, add at least a short description of what this function does.

Reply:
solved in the commit: https://github.com/AgenciaImplementacion/Asistente-LADM_COL/commit/41a6511b624039fa0bb91702dbef2206a0e1bc83